src/features/movies/moviesSlice.js

```diff
@@ -41,9 +39,15 @@ export const selectFetchingStatus = (state) =>
   selectMoviesState(state).fetchingStatus;
 
 export const selectedGenreByIds = (state, genreIds) => {
+  const genres = selectGenres(state);
+
+  if (genres.length > 1) {
     return genreIds.map((genreId) => {
```

Comment:
During making of actor page i've realized that some of cast movies return genre_ids array with ids that don't match any id in our genres
Now when we map over genreIds it always returns array with the same length and we can get an array like this:
`[undefined, 123, undefined]`.
Later when can't map over such array on Tags.
I suggest changing our selectGenresByIds to look like this:
![image](https://user-images.githubusercontent.com/117176251/222831759-361a15fd-1980-493d-9c3b-8dfd2b35666a.png)